service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/Party.java

```diff
@@ -25,6 +27,8 @@
     @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
     public final LocalDate dateOfBirth;
     public final Address address;
+
+    @Valid
     public final EmailAddress email;
     public final Telephone telephoneNumber;
```

Comment:
No need for `@Valid` here also? Perhaps we are only validating a select number of events.